src/common/parallel_for.cpp

```diff
@@ -124,7 +124,7 @@ void ParallelFor::par_for_skip(
         return;
     }
 
-    bitCapInt lowMask = skipMask - 1;
+    bitCapInt lowMask = skipMask - 1UL;
```

Comment:
Don't you want to use that new macro? Also, won't the type of the 1 be promoted correctly?

Reply:
Yes, I want to use the macro. Weirdly, the type will _not_ necessarily be correctly promoted because signed vs. unsigned can take precedence, and this was the whole issue. (It typically will, but it's arguably somewhat ambiguous, depending on the compiler.) We want to remove _all_ ambiguity we can.